package.json

```diff
@@ -46,7 +46,7 @@
     "copy-webpack-plugin": "4.0.1",
     "coveralls": "^2.11.15",
     "css-loader": "^0.26.0",
-    "d3": "^4.4.1",
+    "d3": "^4.9.1",
```

Comment:
Will this  **d3 version 4.x** cause issues with patternfly c3 use of d3 version 3.x. Can they coexist without issue?